content/shmem_bundle_start.tex

```diff
@@ -0,0 +1,35 @@
+\apisummary{
+    Start a communication bundle.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_bundle\_start}@(void);
```

Comment:
why do we need a separate API - I don't see why someone can't use the super-set API with ctx.

Reply:
I don't think we need it, but I included for consistency with fence, quiet, PE-specific quiet, and the RMAs/AMOs.  It's just a convenience routine - I'm happy to remove if a majority find it unnecessary.